src/components/ListItemLayout/styling/ListItemLayout.scss

```
@@ -0,0 +1,135 @@
+  // Outer wrapper: a plain spacing buffer around the interactive container so
+  // the container's focus ring (2px offset) clears neighbouring items and any
+  // overflow/scroll ancestor. Carries the row's minimum height.
+  display: flex;
+  align-items: stretch;
+  width: 100%;
+  min-width: 0;
+  padding-inline: var(--str-chat__spacing-xxs);
+
+  .str-chat__list-item-layout__container {
+    --list-item-padding: var(--str-chat__spacing-xs) var(--str-chat__spacing-sm);
+    display: flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)